docs/source/guides/user/chapters/dependencies.rst

```diff
@@ -3,7 +3,7 @@
 Managing Dependencies
 =====================
 
-.. note:: Test dependencies are supported only on the nrunner runner.
+.. note:: Test dependencies are supported only on the nrunner runner which is the default one.
```

Comment:
We don't have other runner shipped with Avocado, so I believe it's better to omit this line completely at this point.

Reply:
Yes, you are right, it will be better to remove it.